config/routes.rb

```diff
@@ -1,6 +1,15 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  devise_for :users
+  # get 'comments/new'
```

Comment:
- [ ] to keep your repo clean kindly remove all commented code from your project 🙏